package.json

```diff
@@ -23,6 +23,9 @@
       "require": "./dist/index.umd.cjs"
     }
   },
+  "publishConfig": {
+    "access": "public"
+  },
   "engines": {
     "node": ">=18",
     "npm": "USE PNPM",
```

Comment:
I get an "Unsupported Engine" warning when installing the local tarball of this package in a test app: 

![image](https://github.com/user-attachments/assets/32685550-37ae-4355-866b-8aa5c41c3eb3)

We probably need to remove the "USE PNPM" entry (which I assume we want to ensure for development?)

Reply:
I added that "version" to remind myself which package manager to use with this repo. If it's getting in the way of something then we should update it, fine with me.